app/js/components/orderEntry/SearchAndAddOrder.jsx

```diff
@@ -63,7 +87,7 @@ class SearchAndAddOrder extends React.Component {
           </div>
           <br />
         </div>
-      </div>
+      </div> */
```

Comment:
Why do you have this code commented?

Reply:
@Betty-Kebenei, the code is just for demonstrating how to use the tab. So I commented out the bootstrap implementation. I want to first seek the team's approval before deleting.

Reply:
I think we no longer need the bootstrap implementation of the tabs and so it can safely removed.

Reply:
I also think we can remove it.